### DISCUSSION TITLE
文件整理保存源文件名

### DISCUSSION THREAD

**Discussion by ShinJiEDU** [jxxghp/MoviePilot#2531] [问题咨询]
如何设置能做到文件整理的时候保存源文件名，只更改目录？
默认文件整理把大部分信息都删除了。有些时候删除了制作组信息反而不好找字幕。谢谢

**Comment by AboutCXJ** [comment]
配置 MOVIE_RENAME_FORMAT 和 TV_RENAME_FORMAT 就行了

看这个 https://wiki.movie-pilot.org/zh/advanced
自定义重命名那段

**Reply by ShinJiEDU** [reply]
感谢！

**Comment by cng10001** [comment]
![image](https://github.com/user-attachments/assets/f26f805f-bd25-457a-812d-645077eadff4)
**以/分隔多级目录，可调整目录名和文件名格式。**

**源文件名是：original_name： 原文件名（包括文件外缀）**，就放在最后
效果↓
 
![image](https://github.com/user-attachments/assets/57319c86-02fe-4fa0-ab12-5ec09a0f6202)

**Reply by fireflycao** [reply]
{{title}}.{% if original_name %}{{original_name | replace(fileExt, "")}}{% endif %}/{% if original_name %}{{original_name}}{% endif %}
豆瓣名.下载原名无后缀/下载原名带后缀

**Reply by ShinJiEDU** [reply]
感谢！
